From 836769c4a26231f36a48aeadb9e332bd7224c785 Mon Sep 17 00:00:00 2001
From: Andrey Zhizhikin <andrey.zhizhikin@leica-geosystems.com>
Date: Sun, 22 Dec 2019 14:49:57 +0000
Subject: [PATCH 2/2] rtsp: drop incorrect reference to gstreamer-sdp in
 Makefile.am

Rebase following patch from upstream to match NXP version 1.16:

From: Alexander Kanavin <alex.kanavin@gmail.com>
Date: Mon, 26 Oct 2015 17:30:14 +0200
Subject: [PATCH 4/4] rtsp: drop incorrect reference to gstreamer-sdp in
 Makefile.am

Upstream-Status: Pending [review on oe-core maillist]
Signed-off-by: Alexander Kanavin <alex.kanavin@gmail.com>
Signed-off-by: Andrey Zhizhikin <andrey.zhizhikin@leica-geosystems.com>
---
 gst-libs/gst/rtsp/Makefile.am | 1 -
 1 file changed, 1 deletion(-)

diff --git a/gst-libs/gst/rtsp/Makefile.am b/gst-libs/gst/rtsp/Makefile.am
index 13ac1b395..8b03125c7 100644
--- a/gst-libs/gst/rtsp/Makefile.am
+++ b/gst-libs/gst/rtsp/Makefile.am
@@ -82,7 +82,6 @@ GstRtsp-@GST_API_VERSION@.gir: $(INTROSPECTION_SCANNER) libgstrtsp-@GST_API_VERS
 		--pkg gio-2.0 \
 		--pkg gstreamer-@GST_API_VERSION@ \
 		--pkg gstreamer-base-@GST_API_VERSION@ \
-		--pkg gstreamer-sdp-@GST_API_VERSION@ \
 		--pkg-export gstreamer-rtsp-@GST_API_VERSION@ \
 		--add-init-section="$(INTROSPECTION_INIT)" \
 		--output $@ \
-- 
2.17.1

